core/src/main/java/io/questdb/std/ConcurrentHashMap.java

```diff
@@ -1342,6 +1342,28 @@ public V get(CharSequence key) {
         return null;
     }
 
+    public CharSequence getKey(CharSequence key) {
```

Comment:
nit: not used and not commented

Reply:
removed here: https://github.com/questdb/questdb/pull/5159/commits/bb343ca54c27e10b4834e927463d7cd9c7e8a2f1